package.json

```diff
@@ -23,5 +23,9 @@
     "semantic-release": "^8.0.0",
     "standard": "^10.0.3",
     "tap": "^10.7.2"
-  }
-}
+  },
+  "keywords": [],
+  "contributors": [
```

Comment:
so cool! I didn’t know that contributors is a [field in package.json](https://docs.npmjs.com/files/package.json#people-fields-author-contributors)!

Reply:
yeah! Also - I'll be able to fill this out with https://github.com/RichardLitt/name-your-contributors :D

Reply:
very, very cool! That could be a GItHub App all by itself! Keeping contributors up-to-date
You might also be interested in joining forces with https://github.com/kentcdodds/all-contributors/issues/58
I’d love to see that, and happy to help